components/UI/Search/SearchInput.tsx

```diff
@@ -0,0 +1,141 @@
+import React from "react";
+import { TextField } from "@mui/material";
+import { usePlacesWidget } from "react-google-autocomplete";
+import { useMap } from "react-leaflet";
+
+// const placeTypes = [
```

Comment:
can you move this to another file?

Reply:
> can you move this to another file?

For example to where? This the new file I added myself.

Reply:
I marked only the place-types comment line to put it in another file, not the whole file. It is currently obsolete and makes the page unnecessarily crowded. We must keep pages and components clean and readable. That's why I asked. There is no conflict. @emirefek

Reply:
Okay,  going to remove all commented placeTypes thing. If filtering needed devs can follow google api docs: https://developers.google.com/maps/documentation/javascript/supported_types?hl=tr

Reply:
can you add this to readme?

Reply:
Okay updated readme with ENV.VAR warnings and guides. Is there any dev docs? Maybe I can drop the supported type article in there but I'm not quite fan of placing it inside main readme.md.

I'm not joined to Discord groups so I don't have any idea about how you manage stuff internally.